src/cunumeric/cunumeric_c.h

```diff
@@ -28,6 +28,7 @@ enum CuNumericOpCode {
   CUNUMERIC_BINCOUNT,
   CUNUMERIC_CONVERT,
   CUNUMERIC_CONVOLVE,
+  CUNUMERIC_CHOOSE,
```

Comment:
Please move this according to alphabetical order (above CONVERT)

Reply:
Sure, thanks!